src/main/98-variables.tf

```diff
@@ -14,6 +14,26 @@ variable "env" {
   description = "Environment name"
 }
 
+variable "database_subnets" {
+  type        = list(string)
+  description = "Database dedicated private subnets"
+}
+
+variable "eks_control_plane_subnets" {
```

Comment:
Do we actually need to provision control plane subnets? EKS manages the control plane in its own "hidden" VPC

Reply:
Is right that control plane is in a AWS-managed VPC but eks_control_plane_subnets variable controls the subnets where the ENIs for the communication with the control plane are provisioned. Could be useful if should be increased the number of workload subnets without affecting the ENIs subnets.

Reply:
Ok now I get it, sorry I didn't understand the naming of the variable 😄